src/gui/clips/SampleClipView.cpp

```diff
@@ -183,7 +186,7 @@ void SampleClipView::mouseDoubleClickEvent( QMouseEvent * )
 	const QString selectedAudioFile = SampleLoader::openAudioFile();
 
 	if (selectedAudioFile.isEmpty()) { return; }
-	
+
```

Comment:
No need to change this line. As a rule of thumb, try to fix the formatting only on the lines you changed functionally (unless it is specifically a formatting PR).